views_lnurl.py

```diff
@@ -26,25 +26,16 @@ async def lnurl_params(
     request: Request,
     lnpos_id: str,
     payload: str = Query(..., alias="p"),
-    iv: str = Query(...),
 ):
     lnpos = await get_lnpos(lnpos_id)
     if not lnpos:
         raise HTTPException(HTTPStatus.NOT_FOUND, "lnpos not found.")
 
     if len(payload) % 22 != 0:
         raise HTTPException(HTTPStatus.BAD_REQUEST, "Invalid payload length.")
-    if len(iv) != 22:
-        raise HTTPException(HTTPStatus.BAD_REQUEST, "Invalid IV length.")
-    lnpos_payment = await get_lnpos_payment(iv)
-    if lnpos_payment and lnpos_payment.lnpos_id != lnpos_id:
-        raise HTTPException(HTTPStatus.BAD_REQUEST, "Not your payment.")
-    if lnpos_payment and lnpos_payment.payment_hash:
-        raise HTTPException(HTTPStatus.BAD_REQUEST, "Payment already claimed.")
-    if lnpos_payment:
-        raise HTTPException(HTTPStatus.BAD_REQUEST, "Payment already registered.")
```

Comment:
no need to check iv at all, does not matter if multiple invoices are made with the same encryption.